cmd/shell/cmd/client.go

```diff
@@ -1,16 +1,39 @@
 package cmd
 
 import (
+	"context"
+
+	"github.com/NathanBaulch/protoc-gen-cobra/client"
+	"github.com/NathanBaulch/protoc-gen-cobra/naming"
+	"github.com/pactus-project/pactus/www/grpc/basicauth"
 	pb "github.com/pactus-project/pactus/www/grpc/gen/go"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"google.golang.org/grpc"
 )
 
+var Auth = &basicauth.BasicAuth{}
+
 const (
 	defaultServerAddr     = "localhost:50051"
 	defaultResponseFormat = "prettyjson"
 )
 
 func init() {
```

Comment:
One question, Can we add this code inside the main function and remove `init`?
Did you try this?